randovania/games/prime2/generator/base_patches_factory.py

```diff
@@ -1,21 +1,66 @@
 import copy
+import dataclasses
+from collections.abc import Iterable
 from random import Random
-from typing import Iterator
 
 from randovania.game_description.assignment import NodeConfigurationAssociation
 from randovania.game_description.game_description import GameDescription
-from randovania.game_description.game_patches import GamePatches
+from randovania.game_description.game_patches import GamePatches, ElevatorConnection
 from randovania.game_description.requirements.requirement_and import RequirementAnd
 from randovania.game_description.requirements.resource_requirement import ResourceRequirement
 from randovania.game_description.resources import search
 from randovania.game_description.resources.resource_type import ResourceType
+from randovania.game_description.world.area_identifier import AreaIdentifier
 from randovania.game_description.world.configurable_node import ConfigurableNode
 from randovania.game_description.world.dock_node import DockNode
 from randovania.games.prime2.layout.echoes_configuration import EchoesConfiguration
 from randovania.games.prime2.layout.translator_configuration import LayoutTranslatorRequirement
 from randovania.generator.base_patches_factory import (PrimeTrilogyBasePatchesFactory, MissingRng)
 
 
+@dataclasses.dataclass(frozen=True)
+class WorldEntrances:
+    front: AreaIdentifier
+    left: AreaIdentifier
+    right: AreaIdentifier
+
+    @classmethod
+    def create(cls, world, front, left, right):
+        return cls(
+            front=AreaIdentifier(world, front),
+            left=AreaIdentifier(world, left),
+            right=AreaIdentifier(world, right),
+        )
+
+
+WORLDS = [
+    WorldEntrances.create(
+        "Great Temple",
+        front="Temple Transport A",  # Sanc
+        left="Temple Transport B",  # Agon
+        right="Temple Transport C",  # Torvus
+    ),
+    WorldEntrances.create(
+        "Agon Wastes",
+        front="Transport to Temple Grounds",
+        left="Transport to Sanctuary Fortress",
+        right="Transport to Torvus Bog",
+    ),
+    WorldEntrances.create(
+        "Torvus Bog",
+        front="Transport to Temple Grounds",
+        left="Transport to Agon Wastes",
+        right="Transport to Sanctuary Fortress",
+    ),
+    WorldEntrances.create(
+        "Sanctuary Fortress",
+        front="Transport to Temple Grounds",
+        left="Transport to Torvus Bog",
+        right="Transport to Agon Wastes",
```

Comment:
not actually related to this PR but maybe one day we should rename these areas in the db to their descriptive in-game names rather than the vanilla ones